src/request.js

```diff
@@ -8,10 +8,11 @@ class Request {
    */
   constructor(sdk, audienceName) {
     this._audienceName = audienceName;
-    this._sdk = sdk;
     this._axios = axios.create();
+    this._insertHeaders = this._insertHeaders.bind(this);
+    this._sdk = sdk;
 
-    this._axios.interceptors.request.use(this._insertHeaders);
+    this._attachInterceptors(this._axios);
```

Comment:
Is it needed to pass `this._axios` here? It appears that the `_attachInterceptors` function declaration below doesn't take any arguments?

Reply:
👍 Vestigal code smh